ports/srpc/vcpkg.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "srpc",
+  "version": "0.9.3",
```

```suggestion
  "version-semver": "0.9.3",
```
This should be served by [version-semver](https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md#version-semver).